docs/examples.md

```diff
@@ -32,12 +35,32 @@ There are three environment variables that can set custom directory paths for th
  `EUROCROPS_ACQUISITION_DIR`
  : Sets the directory where intermediate data is stored during data acquisition and raw data processing (default `acquisition`).
 
-## Obtaining data for different countries
 
-The ready-to-use EuroCropsML dataset provides preprocessed data for Estonia, Latvia, and Portugal.
-Using the `eurocropsml` package and `eurocropsml-cli` you can use the same processing-pipeline steps also to obtain analogously preprocessed data for other countries. In order to do so, please make sure to download and unzip the necessary [vector data](https://zenodo.org/records/10118572) to into your `EUROCROPS_DATA_DIR` directory. The [NUTS files](https://ec.europa.eu/eurostat/de/web/gisco/geodata/statistical-units/territorial-units-statistics) from Eurostat can be downloaded manually in advance. If not present, they will be downloaded automatically. The structure of the `meta_data` directory should be as follows:
+### Customizing eo-data directory
+
+The `eurocropsml-cli` will, by default, assume that the Sentinel-2 data is located inside a directory called `/eodata`. However, if the directory where the Sentinel-2 is stored is named differently, the `eodata_dir` argument can be utilized in order to change the parent folder of the `.SAFE`-filepaths returned by the EOLab Data Explorer.
+
+```console
+$ eurocropsml-cli acquisition eurocrops get-data +cfg.eodata_dir="personal_eodata_dir"
```

Comment:
The formatting in this section of the Examples docs looks wrong, I think there is some triple backticks missing to close a code block?

Reply:
Ah sorry, seems like  messed this up when resolving the merge conflicts. Should be fixed now :)